decidim-proposals/app/events/decidim/proposals/collaborative_draft_access_rejected_event.rb

```diff
@@ -0,0 +1,29 @@
+# frozen-string_literal: true
+
+module Decidim
+  module Proposals
+    class CollaborativeDraftAccessRejectedEvent < Decidim::Events::SimpleEvent
+      i18n_attributes :requester_nickname, :requester_name, :requester_path, :nickname
+
+      delegate :nickname, :name, to: :requester, prefix: true
+
+      def nickname
```

Comment:
Why are nickname and requester_nickname the same?

Reply:
yes there's no need for it, I'm going to refactor it